File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    a = s.split(" ")
    b = list()
    for l in a:
        c = list(l)
        for i in range(len(c)):
            if i%2==0:
                c[i] = c[i].upper()
            else :
                c[i] = c[i].lower()
        b.append("".join(c))
    return " ".join(b) 

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
import java.util.StringTokenizer;
class Solution {
    public String solution(String s) {
        String answer = "";
        int count = 0;
        String[] str = s.split("");

        for (String s1 : str) {
            if (s1.equals(" ")) {
                answer += " ";
                count = 0;
                continue;
            }
            answer += count % 2 == 0 ? s1.toUpperCase() : s1.toLowerCase();
            count++;
        }

        return answer;
    }
}